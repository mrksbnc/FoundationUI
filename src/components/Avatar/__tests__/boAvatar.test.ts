import {
	AvatarSize,
	AvatarType,
	AvatarVariant,
	BoAvatar,
} from '@/components/Avatar';
import { mount } from '@vue/test-utils';
import { describe, expect, it } from 'vitest';

describe('BoIcon', () => {
	it('renders properly', () => {
		const wrapper = mount(BoAvatar, {
			props: {
				src: undefined,
				initial: undefined,
				size: AvatarSize.MD,
				type: AvatarType.Image,
				initialProps: undefined,
				indicatorProps: undefined,
				variant: AvatarVariant.Rounded,
			},
		});
		expect(wrapper.html()).toMatchSnapshot();
	});
});
