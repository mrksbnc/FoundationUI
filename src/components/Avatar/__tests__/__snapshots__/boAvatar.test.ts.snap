// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BoIcon > renders properly 1`] = `
"<div data-v-3eed3bbf="" class="bo-avatar__container"><img data-v-3eed3bbf="" alt="Avatar" src="/src/assets/img/avatar.png" class="bo-avatar__container__image image rounded md">
  <!--v-if-->
  <!--v-if-->
</div>"
`;
